server/package.json

```diff
@@ -12,20 +12,20 @@
     "db:migrate-create": "cd db && migrate create",
     "watch": "NODE_ENV=development nodemon main.js",
     "start": "NODE_ENV=production node main.js",
-    "test": "NODE_ENV=test jest --runInBand"
+    "test": "NODE_ENV=test jest --runInBand --detectOpenHandles"
```

Comment:
Not sure we should be using `--detectOpenHandles` in all of our tests. Looks like it is primarily used for debugging (see [docs](https://jestjs.io/docs/en/cli)). Also, this arg implies `--runInBand`, so if we're confident that we need `--detectOpenHandles`, we should remove `--runInBand`.

Reply:
I was debugging this and I forgot to remove it because with this command the tests were able to work without warning about open async operations (I guess it’s because massive.js has no close() connection function, and jest takes a while with this command, giving time to massive.js detect an idle connection and close it). So the only way I found to solve this problem is using --forceExit